packages/site/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "devDependencies": {
     "@mineral-ui/button": "^1.0.0",
+    "@mineral-ui/card": "^0.0.1",
```

Comment:
New packages should start out with version 0.0.0.  They will be updated during the first publish.